ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -99,14 +109,16 @@ if [[ -n "${VSPHERE_CONNECTED_LEASED_RESOURCE:-}" ]]; then
   cat >>"${SHARED_DIR}/vsphere_context.sh" <<EOF
 export vsphere_connected_portgroup="${vsphere_connected_portgroup}"
 EOF
+portgroup_list+=("${vsphere_connected_portgroup}")
```

Comment:
Sounds this line missing some leading whitespaces, it should be aligned with the `cat` command

Reply:
done